### PR TITLE
Update add-storage-space.md

### PR DESCRIPTION
**Pull request by msbemba**
The options have changed with NCE and the add-on shows as a subscription and there is no manage add-on option

Per Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10140

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/commerce/add-storage-space.md [hunk 1]
```diff
@@ -74,11 +74,10 @@ If you haven't yet bought extra storage for your subscription, you can do that.
 If you've already bought extra file storage via the **Office 365 Extra File Storage** add-on, you can use these steps to increase or decrease the extra storage space for your subscription. You can reduce the storage to as low as 1 gigabyte. To remove all of the extra storage space, [contact support](../admin/get-help-support.md).
 
 1. In the admin center, go to the **Billing** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=842054" target="_blank">Your products</a> page.
-2. On the **Products** tab, select the subscription that contains the **Office 365 Extra File Storage** add-on.
-3. On the product details page, in the **Add-ons** section, select **Manage add-ons**.
-4. In the **Manage add-ons** pane, from the **Add-on** list, choose **Office 365 Extra File Storage**.
-5. In the **Quantity** text box, enter the number of GBs of storage space that you want for the subscription.
-6. Select **Save**.
+2. On the **Products** tab, select the subscription **Office 365 Extra File Storage** .
+3. On the product details page,  select **Buy Licenses** or **Remove Liceses** based on your requirement.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/add-storage-space.md]:
```suggestion
2. On the **Products** tab, select the subscription **Office 365 Extra File Storage**.
3. On the product details page,  select **Buy Licenses** or **Remove Licenses** based on your requirement.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/commerce/add-storage-space.md [hunk 2]
```diff
@@ -74,11 +74,10 @@ If you haven't yet bought extra storage for your subscription, you can do that.
 If you've already bought extra file storage via the **Office 365 Extra File Storage** add-on, you can use these steps to increase or decrease the extra storage space for your subscription. You can reduce the storage to as low as 1 gigabyte. To remove all of the extra storage space, [contact support](../admin/get-help-support.md).
 
 1. In the admin center, go to the **Billing** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=842054" target="_blank">Your products</a> page.
-2. On the **Products** tab, select the subscription that contains the **Office 365 Extra File Storage** add-on.
-3. On the product details page, in the **Add-ons** section, select **Manage add-ons**.
-4. In the **Manage add-ons** pane, from the **Add-on** list, choose **Office 365 Extra File Storage**.
-5. In the **Quantity** text box, enter the number of GBs of storage space that you want for the subscription.
-6. Select **Save**.
+2. On the **Products** tab, select the subscription **Office 365 Extra File Storage**.
+3. On the product details page,  select **Buy Licenses** or **Remove Licenses** based on your requirement.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/add-storage-space.md]:
```suggestion
3. On the product details page, select **Buy Licenses** or **Remove Licenses** based on your requirement.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by cmcatee-MSFT** [comment]
@yogkumgit , @msbemba  - We can't replace existing procedures for this or any other article with steps for the updated billing experience. We have customers who will still be using the "old" UI for a long time to come and therefore must support both. I'm working through a list of articles like this, so if issues like this get reported, please just send them directly to me. Thanks.

**Comment by yogkumgit** [comment]
@msbemba Please acknowledge, and follow these instruction for the future. Thanks

**Comment by msbemba** [comment]
> @yogkumgit , @msbemba - We can't replace existing procedures for this or any other article with steps for the updated billing experience. We have customers who will still be using the "old" UI for a long time to come and therefore must support both. I'm working through a list of articles like this, so if issues like this get reported, please just send them directly to me. Thanks.

Thank you, @cmcatee-MSFT

**Pull request closed without merging**